core/server/master/src/main/java/alluxio/master/block/BlockMasterWorkerServiceHandler.java

```diff
@@ -61,6 +61,12 @@ public BlockMasterWorkerServiceHandler(BlockMaster blockMaster) {
   @Override
   public void blockHeartbeat(BlockHeartbeatPRequest request,
       StreamObserver<BlockHeartbeatPResponse> responseObserver) {
+    if (LOG.isDebugEnabled()) {
```

Comment:
Is this log level check necessary?

Reply:
I think we avoid one calculation if the log level is above debug